.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I added linter config with prettier.
Config is may be not the best but I didn't wait to spend much time here.